content/download/index.html.haml

```diff
@@ -258,6 +258,12 @@ title: Jenkins download and deployment
   %p{:style=>'margin-bottom:1cm;margin-top:.2cm;'}
     Jenkins Reference Architecture and one-click deployment on Oracle Cloud Infrastructure
 
+%div
+  %a{:href=>'https://github.com/civo/kubernetes-marketplace/tree/master/jenkins'}
+    %img{:src=>'/images/civo.png', :width=>'320'}
```

Comment:
This image is broken on my test installation because the file was committed with upper case 'PNG'.  I'd prefer that the image file be renamed to lower case png so that the site continues to work on case sensitive file systems.
```suggestion
    %img{:src=>'/images/civo.png', :width=>'320'}
```

Reply:
Pushed as 01d2e909a8fc1c4fe4a4be395471893c2b6d1041